cmd/cvdr/main.go

```diff
@@ -46,7 +46,16 @@ func loadInitialConfig() (*cli.Config, error) {
 				return nil, err
 			}
 		} else if errors.Is(err, os.ErrNotExist) {
-			// TODO: Create a new one importing relevant acloud config parameters.
+			// Load
+			acloudConfigPath := cli.ExpandPath("~/.config/acloud/acloud.config")
+			if _, err := os.Stat(acloudConfigPath); err == nil {
+				if err := cli.ImportAcloudConfig(acloudConfigPath, path); err != nil {
```

Comment:
nit: It would be nice obtain user's consent to do this first. Print something like "No user configuration found, would you like to generate it by importing your acloud configuration?".

Reply:
good point, done.